### ISSUE TITLE
Isn't the model sensor-independent?

### ISSUE THREAD

**Issue by study-never-alone** [megvii-research/PMRID#20]
Hi：
      Thanks for this great work. In the paper you said: _Since our proposed denoising network needs to be trained for specific sensors_. So,  we need train sifferent model for different sensor?
      But, in my opinion. Eqn. (10) indicates that the model can be used for any sensor after k-Sigma Transform
![image](https://user-images.githubusercontent.com/26666481/233879266-acefb2f4-f905-45c8-9840-e1f6414f7c50.png)

**Comment by bigeagle** [comment]
Yes, we can use k-sigma transform to use the same model for multiple sensors. 
But the distribution of k, sigma, or the noise profile, are still "defined" by the training data, i.e. the model cannot generalize to untrained  noise profiles. So we cannot claim that this method is sensor-independent.

**Comment by study-never-alone** [comment]
Thanks for your reply.
What do you mean by noise profile? I think the final noise model is only related to f(x*). Can I have a contact information from you, if it's convenient, so that I can consult you in more detail?
